lib/activities/get_extent.js

```diff
@@ -26,12 +31,14 @@ module.exports = function getExtent(uri, callback) {
       return line.match(/Upper Left|Lower Right/);
     })
     .map(function(line) {
-      return line.replace(/^.+\(([-\d\.]+),\s*([-\d\.]+).+$/, "$1 $2").split(" ");
-    })
-    .map(function(pair) {
-      return pair.map(Number);
+      log(line);
+      var m = line.match(re);
+      var x = Number(m[2]);
+      var y = Number(m[3]);
```

Comment:
Optional style / use of (common) weird JS idioms:

```javascript
var m = line.match(re),
    x = +m[2],
    y = +m[3];
```

(`+<var>` coerces `<var>` to a float, same as using `Number`)